includes/core/server/request/class-get-reporting-payment-activity.md

```diff
@@ -0,0 +1,37 @@
+# `Get_Reporting_Payment_Activity` request class
+
+[ℹ️ This document is a part of __WooCommerce Payments Server Requests__](../requests.md)
```

Comment:
When I open https://github.com/Automattic/woocommerce-payments/blob/fix/8742-merchant-timezone-payment-activity/includes/core/server/request/class-get-reporting-payment-activity.md, this link (https://github.com/Automattic/woocommerce-payments/blob/fix/8742-merchant-timezone-payment-activity/includes/core/server/requests.md) leads to a 404.

Reply:
Fixed